packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -123,14 +125,18 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 			}
 		} else if ( isPremium && ! shouldUpgrade && hasPurchasedTheme ) {
 			text = __( 'Purchased on an annual subscription' );
+		} else if ( hasWoopFeature && designIsBundledWithWoo ) {
+			text = __( 'Included in your plan' );
+		} else if ( ! hasWoopFeature && designIsBundledWithWoo ) {
+			text = __( 'Available with Wordpress.com Business' );
```

Comment:
1. There is a typo - it should be "Word**P**ress.com"
2. Having separate conditions here feels a bit weird. I'd prefer code that looked something more like:
```suggestion
const currentSiteCanInstallWoo = currentPlanFeatures.includes( 'woop' ); // Do we have a constant for the `woop` string?
		// ...
		} else if ( designIsBundledWithWoo ) {
			text = currentSiteCanInstallWoo
				? __( 'Included in your plan' )
				: __( 'Available with WordPress.com Business' );
		}
```